### PR TITLE
Add very basic demo template

### PR DESCRIPTION
**Pull request by melanierichards**

### CODE REVIEW

File: demo-template/index.html [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<html lang="en-US">
```
Comment (alrra) [inline demo-template/index.html:2]: Add `<!doctype html>` before this line.

File: README.md [hunk 2]
```diff
@@ -22,9 +22,19 @@ Mention @molant and optionally @melanierichards (for front-end/design review) in
...
+
+#### If you're creating an immersive experience
+
+The demo template is quite permissive; it pretty much just adds a universal header to the top of the page. While we suggest chatting with @melanierichards about design needs, the demos are a nice place to explore different approaches. As long as your demo reflects a cheerful optimism and technical competency, it should be appropriate for this context.
+
+#### If you'd like your demo to be a more simple walkthrough (like @supports or css3filters)
+
+We'd prefer that your demo follow Microsoft design styles more strictly and not reinvent the wheel for simple things. Here's how to do that:
+
+1. Internal folks: chat with @melanierichards
```
Comment (alrra) [inline README.md]: Maybe use [`@melanierichards`](https://github.com/melanierichards)?

File: README.md [hunk 3]
```diff
@@ -22,9 +22,19 @@ Mention @molant and optionally @melanierichards (for front-end/design review) in
...
+2. Un-comment the CSS file reference in the head of the demo-template HTML file
+3. Remove the `demo-header.js` reference in that same HTML file
 
 ## Code of Conduct
```
Comment (alrra) [inline README.md]: Add blank line after this line.

File: README.md [hunk 4]
```diff
@@ -22,9 +22,19 @@ Mention @molant and optionally @melanierichards (for front-end/design review) in
...
+#### If you're creating an immersive experience
+
+The demo template is quite permissive; it pretty much just adds a universal header to the top of the page. While we suggest chatting with @melanierichards about design needs, the demos are a nice place to explore different approaches. As long as your demo reflects a cheerful optimism and technical competency, it should be appropriate for this context.
+
+#### If you'd like your demo to be a more simple walkthrough (like @supports or css3filters)
+
+We'd prefer that your demo follow Microsoft design styles more strictly and not reinvent the wheel for simple things. Here's how to do that:
+
+1. Internal folks: chat with @melanierichards
+2. Un-comment the CSS file reference in the head of the demo-template HTML file
```
Comment (alrra) [inline README.md]: **After this is merged**, we should change this line to include a link to that CSS file reference (e.g.: [here's how that link looks like for @melanierichards's fork](https://github.com/melanierichards/Demos/blob/a659edb0152f0ac1abace71ae1a488d533a47691/demo-template/index.html#L10)).

File: README.md [hunk 5]
```diff
@@ -22,9 +22,19 @@ Mention @molant and optionally @melanierichards (for front-end/design review) in
...
+3. Remove the `demo-header.js` reference in that same HTML file
```
Comment (alrra) [inline README.md]: **After this is merged**, we should change this line to include a link to the `demo-header.js`reference.

**Review by alrra — CHANGES_REQUESTED** [review]

**Comment by melanierichards** [comment]
@alrra I made all changes, except for the "after this is merged" comments

**Review by alrra — APPROVED** [review]

**Comment by alrra** [comment]
> I made all changes, except for the "after this is merged" comments

@melanierichards Great, thanks! 💜

**Comment by molant** [comment]
@melanierichards @sarvaje I think we should add the `config.json` to the demo template if people are going to write their own styles (or change the default behavior we have in the site?)
What do you think?

**Comment by melanierichards** [comment]
@molant doth this PR please thee now?

**Pull request merged**